README.md

```diff
@@ -508,12 +512,12 @@ Speculative: Adding noise to the conversion event itself
 --------------------------------------------------------
 
 Another way to add privacy to this system is to add noise not only to
-the [reported conversion metadata value](https://github.com/csharrison/conversion-measurement-api#metadata-limits-and-noise),
+the [reported conversion data value](https://github.com/csharrison/conversion-measurement-api#metadata-limits-and-noise),
```

Comment:
s/metadata/data in the ref part of the URL (while you're at it you can change the URL from an absolute one to a relative one since this is wrong (we're in WICG now)

Reply:
Right - fixed, changed to relative URL.